src/models/user.ts

```diff
@@ -0,0 +1,35 @@
+import {
+  Entity,
+  model,
+  property
+} from "@loopback/repository";
+
+@model()
+export class User extends Entity {
+  @property({
+    description: "The ID for a user.",
+    id: true,
+    type: "number"
+  })
+  id: number;
+  @property({
+    description: "The username for a user.",
+    required: true,
+    type: "string"
+  })
+  username: string;
+  @property({
+    description: "The number of times a user has visited the site.",
+    required: false,
+    type: "number",
+  })
+  visited: number;
+  
+  getId(): number {
```

Comment:
I believe this is provided by `Entity` for you, you should not need to duplicate it here.